### PR TITLE
New/gum 29 location autocomplete

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

File: src/components/autocomplete-select/autocomplete-select.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  GestureResponderEvent,
+  LayoutRectangle,
+  ScrollView,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+import { Portal, TouchableRipple } from 'react-native-paper';
+import _ from 'lodash';
```
Comment (yevhenprybytok) [inline src/components/autocomplete-select/autocomplete-select.tsx]: Let's use lodash-es and also let's import functions as follow `import { isFunction } from 'lodash-es';`

File: src/components/form/form-google-location-select/form-google-location-select.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      return;
+    }
+
+    if (!values[name] || !values[name].length) {
+      setOptions([]);
+      return;
+    }
+
+    setOptionsLoading(true);
+    setOptions(await googlePlacesApi.predictAutocomplete(values[name]));
```
Comment (yevhenprybytok) [inline src/components/form/form-google-location-select/form-google-location-select.tsx]:
```suggestion
    const options = await googlePlacesApi.predictAutocomplete(values[name]);
    setOptions(options);
```

File: src/components/autocomplete-select/styles.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    backgroundColor: AppColor.INVISIBLE,
+  },
+  option: {
+    backgroundColor: AppColor.WHITE,
+    paddingVertical: 10,
+    paddingHorizontal: 20,
+  },
+  fill: {
+    width: '100%',
+    height: '100%',
```
Comment (katyakostikova) [inline src/components/autocomplete-select/styles.ts]:
```suggestion
  flex: 1
```

**Review by yevhenprybytok — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -39,6 +39,7 @@
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
     "@types/jest": "^26.0.23",
+    "@types/lodash-es": "^4.17.6",
```
Comment (yevhenprybytok) [inline package.json:43]: Where is the dev dependency?

**Review by katyakostikova — APPROVED** [review]

**Pull request merged**